helm/rbac-app/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: 0.0.1
+annotations:
+  application.giantswarm.io/team: "rainbow"
+  config.giantswarm.io/version: 1.x.x
+name: rbac-app
+description: A Helm chart for rbac
```

Comment:
I know a few people in the company that won't be happy about this generic description